src/components/Container.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import { MessageList } from "./MessageList.jsx";
+import { NumberOfLikes } from "./NumberOfLikes.jsx";
+import { MessageInput } from "./MessageInput.jsx";
+import { Header } from "./Header.jsx";
+
+export const Container = () => {
+  const [messageData, setMessageData] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [totalNumberOfLikes, setTotalNumberOfLikes] = useState(() => {
+    const storedLikes = localStorage.getItem("numberOfLikes");
+    return storedLikes ? JSON.parse(storedLikes) : "";
```

Comment:
maybe this needs to be a number 0 instead of a string? each time I press it the 1 goes on the end so it says "You have liked 1 happy thoughts", "You have liked 11 happy thoughts", "You have liked 111 happy thoughts.° instead of saying 1, 2, 3 and so on